App/Models/Task.cs

```diff
@@ -2,16 +2,20 @@ namespace App.Models
 {
     public class Task
     {
-        public static readonly long DEFAULT_ID = -1;
+        public const long DEFAULT_ID = -1;
+        public const bool DONE = true;
```

Comment:
I am not sure the `DONE` and `NOT_DONE` constants are at all necessary. I understand that they were added to get rid of "magic numbers", similarly to how `DEFAULT_ID` was introduced. But interacting with a field like `IsDone` using boolean values is very intuitive, and aliasing `true` and `false` to `DONE` and `NOT_DONE` doesn't provide any additional clarity.


I think these constants can be safely removed, as it's unlikely they are gonna be used very often.

Reply:
You're thinking exactly right, I added them to get rid of the magic numbers. I'll remove it for now.
But I still think that the is_done field should be changed to status and stored as an enum

Reply:
And the call to `repository.UpdateTaskStatus(taskId, Task.DONE);`
is more understandable than `repository.UpdateTaskStatus(taskId, true);`

Reply:
Agreed